.github/workflows/deploy.yml

```diff
@@ -43,4 +43,4 @@ jobs:
       with:
         push: true
         platforms: linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64/v8
-        tags: weejewel/wg-easy:latest, weejewel/wg-easy:${{ env.RELEASE }}
+        tags: jdeath/wg-easy:latest, jdeath/wg-easy:${{ env.RELEASE }}
```

Comment:
?

Reply:
This was so I could use and test it. I didn't know how to remove this file...